### PR TITLE
Display names has been changed

### PR DESCRIPTION
**Pull request by recena**
@reviewbybees specially @rtyler and @daniel-beck

### CODE REVIEW

**Comment by kwhetstone** [comment]
:+1: I like this (I'll let Tyler and Daniel weigh in for the bees)

**Comment by HRMPW** [comment]
:bee:

**Comment by HRMPW** [comment]
"Maintained" or "Contributed" both sound fine for me but let's be consistent. If we're going to use "Maintained by CloudBees" we should use the same phrase on all of them.

**Comment by jglick** [comment]
:bee:, long overdue

**Comment by stephenc** [comment]
:bee: w00t!

**Comment by oleg-nenashev** [comment]
:bee: . I won't need to explain it during demos anymore :+1:

**Comment by recena** [comment]
@reviewbybees done

File: pom.xml [hunk 1]
```diff
@@ -12,9 +12,9 @@
   <version>5.4-SNAPSHOT</version>
   <packaging>hpi</packaging>
 
-  <name>CloudBees Folders Plugin</name>
+  <name>Folders Plugin</name>
   <description>This plugin allows users to create "folders" to organize jobs. Users can define custom taxonomies (like
-    by project type, organization type etc). Folders are nestable and you can define views within folders.
+    by project type, organization type etc). Folders are nestable and you can define views within folders. Maintained by CloudBees, Inc.
   </description>
   <url>https://wiki.jenkins-ci.org/display/JENKINS/CloudBees+Folders+Plugin</url>
```
Comment (jglick) [inline pom.xml]: @recena do you not plan to also rename the wiki pages?

Comment (recena) [inline pom.xml]: @jglick Yes, but after to Jenkins 2.0. Is it ok for you?

Comment (jglick) [inline pom.xml]: You mean, because you are too busy now? That is fine, just checking if it was planned.

Comment (recena) [inline pom.xml]: @jglick Exactly. I prefer to do all pending things at the same time: cloudbees-disk-usage-simple-plugin, cloudbees-folder-plugin, etc...

**Pull request merged**